modules/virt-about-services.adoc

```diff
@@ -8,7 +8,7 @@
 
 A Kubernetes _service_ exposes network access for clients to an application running on a set of pods. Services offer abstraction, load balancing, and, in the case of NodePort and LoadBalancer, exposure to the outside world. 
 
-Services can be exposed in different ways by specifying a `spec.type` in the `Service` object:
+Services can be exposed in the *Details* tab of the web console or by specifying a `spec.type` in the `Service` object:
```

Comment:
Should we change "the web console" to "the virtual machine"? *Details* tab of the web console does not make sense.

Reply:
@gouyang I agree with this since you click on Virt->VirtualMachines on the left, then pick a VM instance, and go to the Details tab. So I can make this change.

My only other question would be wording - should it be `Details tab of the VirtualMachine` or `Details tab of the virtual machine`?

Reply:
While looking at other pages in the virt doc, it's using `VirtualMachine`, so let' using `VirtualMachine` here.

Reply:
... **on** the *Details* tab **on** the *VirtualMachine details* page of the web console. See [IBM style guide: UI elements](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui), which describes the correct prepositions. This can also be expressed as *VirtualMachine details* -> *Details* tab of the web console.

Reply:
Thanks for the input. I went ahead and pushed a new change using:
`*VirtualMachine details* -> *Details* tab of the web console`

Let me know if there are any problems!